engine/pom.xml

```diff
@@ -218,8 +218,10 @@
                             org.hibernate.search.query;version="${project.version}",
                             org.hibernate.search.query.collector;version="${project.version}",
                             org.hibernate.search.query.dsl;version="${project.version}",
+                            org.hibernate.search.query.dsl.sort;version="${project.version}",
                             org.hibernate.search.query.engine;version="${project.version}",
                             org.hibernate.search.query.engine.spi;version="${project.version}",
+                            org.hibernate.search.query.engine.spi.impl;version="${project.version}",
```

Comment:
@gsmet The reason I answered you this way is that the code in this `spi.impl` package is actually an implementation of an interface in the `spi` package. It's not just random internal code.
But yes, given it's not intended for external use anyway, we could move it to `engine.impl`.